src/Commands/pm/PmCommands.php

```diff
@@ -235,6 +235,7 @@ public function validateUninstall(CommandData $commandData): void
     ])]
     #[CLI\DefaultTableFields(fields: ['package', 'display_name', 'status', 'version'])]
     #[CLI\FilterDefaultField(field: 'display_name')]
+    #[CLI\TableEmptyMessage(message: 'There are no items to list')]
```

Comment:
Run `drush pm:list --filter=package=Core` to see a list of core modules; run `drush pm:list --filter=package=Foo` to see the empty message.